apps/ledger-live-mobile/src/config/urls.tsx

```diff
@@ -83,8 +83,6 @@ export const urls = {
   },
   celoStakingRewards:
     "https://support.ledger.com/hc/en-us/articles/360020499920-Celo-CELO-?utm_source=ledger_live_mobile&utm_medium=self_referral&utm_content=celo&docs=true",
-  osmosisStakingRewards:
-    "https://support.ledger.com/hc/en-us/sections/4404369637521-Crypto-assets?docs=true",
```

Comment:
fixed. I removed the "osmosisStakingRewards" in urls.tsx because I've moved them in "chain" folder.
We have one url per chain.
@henrily-ledger will generaliz this logic in LLD and LLM

Reply:
Fixed now
I use "stakingDocUrl" in chain folder to cover all chains generically now